tests/security/389ds/tls_389ds_sssd_client.pm

```diff
@@ -0,0 +1,75 @@
+# Copyright (C) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Implement & Integrate 389ds + sssd test case into openQA,
+#          This test module covers the sssd client tests
+#
+# Maintainer: rfan1 <richard.fan@suse.com>
+# Tags: poo#88513, tc#1768672
+
+use base 'consoletest';
+use testapi;
+use strict;
+use warnings;
+use utils;
+use lockapi;
+use Utils::Systemd qw(systemctl disable_and_stop_service);
+
+sub run {
+    select_console("root-console");
+
+    my $remote_ip   = '10.0.2.101';
```

Comment:
I feel I am a bit pick nit. Since this is a Client/Server testing. So I'd like to use server_ip and client_ip, server_name and client_name here; but it is totally fine without changing. :)